### ISSUE TITLE
Support Packet Loss Statistics

### ISSUE THREAD

**Issue by samgomena** [alessandromaggio/pythonping#58]
**Is your feature request related to a problem? Please describe.**
I have a use case for determining the packet loss percentage from a ping request. I don't believe this library supports that.

**Describe the solution you'd like**
I would like to be able to determine the percentage of packets lost in a ping request. Similar to how a successful ping can be determined:

```python
# Supported
ping("localhost").success()

# Not supported
pint("localhost").packet_loss()
```

**Describe alternatives you've considered**
I've forked the library and have what I think is a naive implementation:
```python
# In executor.py > ResponseList
def packet_loss(self):
    success_list = [resp.success for resp in self._responses]
    return (success_list.count(False) / len(success_list)) * 100
```

**Additional context**
I noticed there was a [comment on another issue](https://github.com/alessandromaggio/pythonping/issues/16#issuecomment-566869147) where someone mentioned they were also interested in determining packet loss. If this is a feature that you would be willing to add to the library, I'd be more than happy to submit a PR for it!

**Comment by alessandromaggio** [comment]
This is an interesting that would definitely be beneficial to a lot of people. If you make a PR for it I'll be glad to integrate it in the next release.

Please let me make some considerations on the implementation:

1. The rate must be normalized in a 0-1 range (0% loss represented as 0.0, 100% loss as 1.0)
2. Instead of computing a normal average by interating again on all the data after the ping results come in, compute a running average *while* the data comes in, so that reading the packet_loss() property will only return that value without having to do additional computation.

Thank you!

**Comment by alessandromaggio** [comment]
Included in release 1.0.15, already out. Thank you @samgomena!
